yaml_edit/test/append_test.dart

```diff
@@ -147,6 +169,13 @@ a:
       expectYamlBuilderValue(doc, [0, 1, 2, 3]);
     });
 
+    test('null value', () {
+      final doc = YamlEditor('[0, 1, 2]');
+      doc.appendToList([], null);
+      expect(doc.toString(), equals('[0, 1, 2, null]'));
+      expectYamlBuilderValue(doc, [0, 1, 2, null]);
+    });
+
```

Comment:
Tests that explicitly cover appending null values to list

Reply:
I think it's fair that `wrapAsYamlNode` can take null... and then it's also fair that `value` in `insertIntoList` (and friends), is null-able... because those values are automatically wrapped with `wrapAsYamlNode`.